### ISSUE TITLE
Feature Request: Hide macOS traffic lights

### ISSUE THREAD

**Issue by frankreporting** [mgmeyers/obsidian-electron-window-tweaker#7]
This plugin would dovetail really nicely with [kepano's Hider](https://github.com/kepano/obsidian-hider) if you could also hide the macOS traffic lights except when hovering. This would allow you to work with a super minimalist interface on par with the [Ulysses writing app](https://ulysses.app).

The Electron docs [appear to suggest this is possible](https://www.electronjs.org/docs/latest/tutorial/window-customization#control-the-traffic-lights-macos):

```
const { BrowserWindow } = require('electron')
const win = new BrowserWindow({ titleBarStyle: 'customButtonsOnHover' })
```

In any case, very cool plugin!

**Comment by chrisgrieser** [comment]
You can hide the traffic lights already by assigning negativ values. `-70` for Traffic Light X completely hides the traffic light for me

**Comment by frankreporting** [comment]
> You can hide the traffic lights already by assigning negativ values. `-70` for Traffic Light X completely hides the traffic light for me

That's true. I did that for a while and then thought better of it. The drawback is that you lose them completely. 

Ideally they'd disappear until you hover, much like the hidden elements in Minimal theme or Hider plugin. 

I admit this is a highly particular use case. Just throwing it out there! I actually am trying to learn plugin development so maybe I can take a stab eventually and do a PR.
